doc/source/whatsnew/v0.25.0.rst

```diff
@@ -395,6 +395,7 @@ Reshaping
 - Bug in :func:`concat` where the resulting ``freq`` of two :class:`DatetimeIndex` with the same ``freq`` would be dropped (:issue:`3232`).
 - Bug in :func:`merge` where merging with equivalent Categorical dtypes was raising an error (:issue:`22501`)
 - Bug in :class:`DataFrame` constructor when passing non-empty tuples would cause a segmentation fault (:issue:`25691`)
+- Bug in :func:`pandas.cut` where large bins could raise an error due to an overflow (:issue:`26045`)
```

Comment:
Can you make this a little more precise: "could raise an error" --> "could incorrectly raise an error"